package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A system that allows efficient, cost-effective management of our library.",
   "main": "index.js",
   "scripts": {
-    "test": "cross-env NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout=10000",
+    "test": "NODE_ENV=test nyc --reporter=html --reporter=text mocha --timeout=10000 --exit",
```

Comment:
Why did you get rid of the `cross-env` package?

Reply:
not intentional I was using to test why something wasn't working and I forgot to add it back